Part one of the assignment

Zelphy712 opened the pull request:
Bonus: changed color and font of the text, made the text bounce around the screen

Zelphy712 committed:
Part one of the assignment

Bonus: changed color and font of the text, made the text bounce around the screen

SamBumgardner reviewed (commented):
Looks good to me. Will check in again when part 2 is done.

Zelphy712 committed:
Added a hero class

made three modes for the movement, bounce, toroidal, and mouse follow mode. they can be switched with the space bar. randomization was skipped for the three modes but might be readded later

SamBumgardner reviewed (approved):
Gotta say, the bounce mode stuff is pretty cool. Nice work!

Feel free to merge when ready.
